tests/Http/SignupTest.php

```diff
@@ -590,7 +590,7 @@ public function testSignupIndexWithIncludedUser()
      */
     public function testSignupsIndexAsAdminWithFilters()
     {
-        $northstarId = $this->faker->northstar_id;
+        $northstarId = $this->faker->unique()->northstar_id;
```

Comment:
Is this because the referral ID would be set to the same ID as the post's user?

Reply:
There's the slight chance it could be, because our northstar_id faker selects a random element from a list of valid dev Northstar ID's. This is one of those tests that will randomly fail and then pass on a re-run, this should hopefully fix at least this one (there's a few).